packages/utils/types/organizations.ts

```diff
@@ -42,7 +42,7 @@ export type MembershipMemberSettingFormType = {
 // ROLES BASED ORGANIZATION FORM TYPES
 
 export type RolesSettingFormType = {
-  roles: { name: string; color: string }[];
+  roles: { name: string; color: string, features: string[] }[];
```

Comment:
can we rename "feature" to "ressource" (or something else) in the code? we already have the concept of `NetworkFeature` and I'm afraid this naming will cause confusion